src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -421,14 +421,18 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         manager.register_pass<ov::pass::ConvertPad12ToPad1, false>();
         manager.register_pass<DecomposeReduceForScalarOutput>();
 
-        precisions_map int_convert_precision_map {
-                {ov::element::i64, ov::element::i32},
-                {ov::element::u64, ov::element::i32},
-                {ov::element::u16, ov::element::i32},
-                {ov::element::u32, ov::element::i32},
-                {ov::element::boolean, ov::element::u8},
-                {ov::element::i4, ov::element::i8},
-                {ov::element::u4, ov::element::u8},
+        precisions_map int_convert_precision_map{
+            {ov::element::i64, ov::element::i32},
+            {ov::element::u64, ov::element::i32},
+            {ov::element::i16,
+             ov::element::i32},  //< Workaround for supporting 16 by Elementwise ops... i16 is converted here to i32, to
```

Comment:
It is really converted back to i16 somewhere? Which part is doing that?

Reply:
If I recall correctly, it is Datatype EltwiseKernelBase::GetAccumulatorType(const eltwise_params &params)  in case of elementwise, but I am not sure at this moment.

Reply:
UPDATE: I have check it and it turns that this comment is wrong: inputs to elementwise are i32 - will remove this comment.

Reply:
Fixed.